.github/dependabot.yml

```
@@ -9,3 +9,30 @@ updates:
       - rebase
     commit-message:
       prefix: "rebase"
+    ignore:
+      # k8s.io/kubernetes prevents auto-updating these
+      - dependency-name: "k8s.io/api"
```

```suggestion
      - dependency-name: "k8s.io/*"
```

This will work too?

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#specifying-dependencies-and-versions-to-ignore

---

it might work, but we use `k8s.io/klog` and `k8s.io/utils` as well (maybe more?), and those can be updated separately

---

Okay 👍

---

kind of agree to both, ideally its good to get all the kube dependencies ( which include klog , utils..etc) fetched in single update ,  so `*` make sense but I am not sure it works.  We can try this model too.

---

@humblec , `k8s.io/klog` and `k8s.io/utils` are versioned independently of `k8s.io/kubernetes`, so updating them together with `k8s.io/kubernetes` is not particularly  useful.